Examples/ExamplesApp/Examples.swift

```diff
@@ -39,7 +39,7 @@ extension ExampleList {
             AnyExample("Bookmarks", content: BookmarksExampleView()),
             AnyExample("Compass", content: CompassExampleView()),
             AnyExample("Floor Filter", content: FloorFilterExampleView()),
-            AnyExample("Form", content: FormExampleView()),
+            AnyExample("FormView", content: FormViewExampleView()),
```

Comment:
Seems like it'd be more correct to use the component's symbol name here (no space).

Should the other entries be changed to match? I can take care of that separately on `v.next`.